packages/components-vue/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Should this file have the word `lib` in its name?

Reply:
We dont have it in react and angular, would keep it in sync?